requirements/runtime.txt

```diff
@@ -3,5 +3,6 @@
 
 https://github.com/level12/keg/zipball/master
 arrow
+wtforms_components
```

Comment:
KE already had [a direct dependency on this](https://github.com/level12/keg-elements/blob/master/keg_elements/forms/__init__.py#L17). I'll fix the name.